packages/groff/build.sh

```diff
@@ -5,7 +5,13 @@ TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=1.22.4
```

Comment:
https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#rebuilding-package-with-no-version-change

Reply:
Just need to add?

Reply:
Every package rebuild within same version requires **a revision bump**. If revision variable is absent, you need to add it. If still have doubts how to bump package correctly, there is a script `./scripts/bin/revbump` (use like `./scripts/bin/revbump groff`).